examples/va/nfv/ovs-dpdk/edpm/nova_ovs_dpdk.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# CHANGEME:
+#
+# - Change cpu-pinning-nova ConfigMap's "data['04-cpu-pinning-nova.conf']" to use CPU sets
+#   proper for your environment
+#
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ovs-dpdk-cpu-pinning-nova
+data:
+  04-cpu-pinning-nova.conf: |
+    [compute]
+    cpu_shared_set = 0-3,24-27 # CHANGEME
+    cpu_dedicated_set = 8-23,32-47 # CHANGEME
+    reserved_host_memory_mb = 4096
```

Comment:
This format is not correct as per information received from the compute team.  Please see this for reference: https://github.com/openstack-k8s-operators/architecture/blob/b18ae4a41c1bf27d58b15a17d8b7b95db4931ed2/examples/va/nfv/sriov/edpm/values.yaml#L137-L146

Reply:
@abays Thanks.
reserved_huge_pages config needed?
Can you please corresponding parameters in tripleo?

Reply:
> This format is not correct as per information received from the compute team. Please see this for reference:
> 
> https://github.com/openstack-k8s-operators/architecture/blob/b18ae4a41c1bf27d58b15a17d8b7b95db4931ed2/examples/va/nfv/sriov/edpm/values.yaml#L137-L146


Ack. reserved_host_memory_mb is part of DEFAULT section only. will update it.

Reply:
Unfortunately I don't know what the related OOO parameters are.  @SeanMooney helped me understand what the proper approach is here [1].

[1] https://github.com/openstack-k8s-operators/architecture/pull/76#discussion_r1456407995